coordinator/src/main/scala/filodb.coordinator/queryplanner/MultiPartitionPlanner.scala

```diff
@@ -120,40 +114,40 @@ class MultiPartitionPlanner(partitionLocationProvider: PartitionLocationProvider
       // Binary Join supported only fro single partition now
       if (partitions.forall(_.partitionName.equals((partitionName)))) {
         if (partitionName.equals(localPartitionName)) localPartitionPlanner.materialize(logicalPlan, qContext)
-        else PromQlExec(qContext, InProcessPlanDispatcher, dataset.ref,
-          generateRemoteExecParams(qContext, logicalPlanTime.startMs / 1000, logicalPlanTime.endMs / 1000,
-            partitions.head.endPoint))
+        else PromQlRemoteExec(qContext, InProcessPlanDispatcher, dataset.ref,
+          generateRemoteExecParams(queryParams, partitions.head.endPoint, queryParams.startSecs * 1000, queryParams.endSecs * 1000))
```

Comment:
Instant queries can also have binary join so we need to add `logicalPlan.isInstanceOf[RawSeriesLikePlan]`

Reply:
done